### ISSUE TITLE
RetroAchievements don't work if a game is set to ask for emulator on startup

### ISSUE THREAD

**Issue by LordArchantos** [Lacro59/playnite-successstory-plugin#464]
For convenience, I have set up both Steam and regular versions of RetroArch through Playnite, as well as one extra profile to run with achievements off. So when I scan games for RetroArch, I set the scanner to ask to choose the emulator at startup.
<img width="195" alt="image" src="https://github.com/Lacro59/playnite-successstory-plugin/assets/45307447/173cddae-b4f7-4607-adbf-0c70ba7163ea">

I've found that currently SuccessStory does not work if the only play action has the emulator set to "Choose at startup". The plugin doesn't show up in the context menu for these games either. The games have platforms assigned, which are associated in the SuccessStory settings as well.

![image](https://github.com/Lacro59/playnite-successstory-plugin/assets/45307447/032fb42d-5b30-42bc-a0be-5fcf9c1a1c50)

Additionally, any games set up like this, that I had previously scanned achievements for, now lose them if I e.g. add a game to favourites and then from the Extensions -menu download plugin data for Favourites.

![image](https://github.com/Lacro59/playnite-successstory-plugin/assets/45307447/19ddf6c8-04fa-4471-a40f-0a7e571e4a3f)

TL;DR: Currently the plugin only works correctly when a game has at least one action with an emulator option other than "choose on startup".
